extend_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+. util.sh
+
+if [[ $# -ne 2 ]] || [[ ! -e "$1" ]] || [[ ! -e "$2" ]] ; then
+	echo "Usage: $0 path/to/bzImage path/to/initrd"
+	exit
+fi
+
+# see https://www.kernel.org/doc/html/latest/x86/boot.html#details-of-harder-fileds
```

Comment:
I can't work out if the typos in this Linux's documentation are deliberate.  Normally, deliberate typos tend to be less subtle.